static/js/github-app-creation.js

```diff
@@ -0,0 +1,40 @@
+/* globals $, AP */
```

Comment:
This file needs to be renamed to jira-app-creation. At this point in the flow we are still 'in jira'. We switch to prepending GH when we open a new tab to navigate our of Jira.